Mathlib/RepresentationTheory/Homological/GroupCohomology/LowDegree.lean

```diff
@@ -563,52 +703,62 @@ variable {k G A : Type u} [CommRing k] [Group G] [AddCommGroup A] [Module k A]
 `A` induced by the `DistribMulAction`. -/
 @[simps]
-def oneCocyclesOfIsOneCocycle {f : G → A} (hf : IsOneCocycle f) :
-    oneCocycles (Rep.ofDistribMulAction k G A) :=
-  ⟨f, (mem_oneCocycles_iff (A := Rep.ofDistribMulAction k G A) f).2 hf⟩
+def cocyclesOfIsCocycle₁ {f : G → A} (hf : IsCocycle₁ f) :
+    cocycles₁ (Rep.ofDistribMulAction k G A) :=
+  ⟨f, (mem_cocycles₁_iff (A := Rep.ofDistribMulAction k G A) f).2 hf⟩
+
+@[deprecated (since := "2025-06-25")] alias oneCocyclesOfIsOneCocycle := cocyclesOfIsCocycle₁
 
-theorem isOneCocycle_of_mem_oneCocycles
-    (f : G → A) (hf : f ∈ oneCocycles (Rep.ofDistribMulAction k G A)) :
-    IsOneCocycle f :=
-  fun _ _ => (mem_oneCocycles_iff (A := Rep.ofDistribMulAction k G A) f).1 hf _ _
+theorem isCocycle₁_of_mem_cocycles₁
+    (f : G → A) (hf : f ∈ cocycles₁ (Rep.ofDistribMulAction k G A)) :
+    IsCocycle₁ f :=
+  fun _ _ => (mem_cocycles₁_iff (A := Rep.ofDistribMulAction k G A) f).1 hf _ _
 
 /-- Given a `k`-module `A` with a compatible `DistribMulAction` of `G`, and a function
 `f : G → A` satisfying the 1-coboundary condition, produces a 1-coboundary for the representation
 on `A` induced by the `DistribMulAction`. -/
 @[simps]
-def oneCoboundariesOfIsOneCoboundary {f : G → A} (hf : IsOneCoboundary f) :
-    oneCoboundaries (Rep.ofDistribMulAction k G A) :=
+def coboundariesOfIsCoboundary₁ {f : G → A} (hf : IsCoboundary₁ f) :
+    coboundaries₁ (Rep.ofDistribMulAction k G A) :=
   ⟨f, hf.choose, funext hf.choose_spec⟩
 
-theorem isOneCoboundary_of_mem_oneCoboundaries
-    (f : G → A) (hf : f ∈ oneCoboundaries (Rep.ofDistribMulAction k G A)) :
-    IsOneCoboundary f := by
+@[deprecated (since := "2025-06-25")]
+alias oneCoboundariesOfIsOneCoboundary := coboundariesOfIsCoboundary₁
+
+theorem isCoboundary₁_of_mem_coboundaries₁
+    (f : G → A) (hf : f ∈ coboundaries₁ (Rep.ofDistribMulAction k G A)) :
+    IsCoboundary₁ f := by
   rcases hf with ⟨a, rfl⟩
   exact ⟨a, fun _ => rfl⟩
 
 /-- Given a `k`-module `A` with a compatible `DistribMulAction` of `G`, and a function
 `f : G × G → A` satisfying the 2-cocycle condition, produces a 2-cocycle for the representation on
 `A` induced by the `DistribMulAction`. -/
 @[simps]
-def twoCocyclesOfIsTwoCocycle {f : G × G → A} (hf : IsTwoCocycle f) :
-    twoCocycles (Rep.ofDistribMulAction k G A) :=
-  ⟨f, (mem_twoCocycles_iff (A := Rep.ofDistribMulAction k G A) f).2 hf⟩
+def cocyclesOfIsCocycle₂ {f : G × G → A} (hf : IsCocycle₂ f) :
+    cocycles₂ (Rep.ofDistribMulAction k G A) :=
+  ⟨f, (mem_cocycles₂_iff (A := Rep.ofDistribMulAction k G A) f).2 hf⟩
 
-theorem isTwoCocycle_of_mem_twoCocycles
-    (f : G × G → A) (hf : f ∈ twoCocycles (Rep.ofDistribMulAction k G A)) :
-    IsTwoCocycle f := (mem_twoCocycles_iff (A := Rep.ofDistribMulAction k G A) f).1 hf
+@[deprecated (since := "2025-06-25")] alias twoCocyclesOfIsTwoCocycle := cocyclesOfIsCocycle₂
+
+theorem isCocycle₂_of_mem_cocycles₂
```

Comment:
And this one? I guess it's kind of helpful to get these right because I think the class field theory repo is going to mega-break when this is merged and it will be much easier to fix if all the deprecations are there.